tachyon/math/polynomials/univariate/mixed_radix_evaluation_domain.h

```diff
@@ -90,6 +91,7 @@ class MixedRadixEvaluationDomain
   }
 
   CONSTEXPR_IF_NOT_OPENMP void DoFFT(Evals& evals) const override {
+    TRACE_EVENT("EvaluationDomain", "MixedRadixEvaluationDomain::DoFFT");
```

Comment:
According to your rule that adds class name only to the public method, then shouldn't it be `DoFFT`?

Reply:
The flow here is `UnivariateEvaluationDomain::FFT` -> `MixedRadixEvaluationDomain::DoFFT` it is hard to differentiate without the `MixedRadixEvaluationDomain` namespace.